js/map_utils.js

```diff
@@ -163,6 +163,11 @@ function updateView(newView, fireAnalytics) {
   unhighlightCircle();
   updateLegendTextToView();
   
+  // only reset stroke when zooming back out
+  if(activeView === "USA") {
+    resetCircleStroke();
+  }
```

Comment:
is there a reason this `if` block needs to occur before lines 171-173 (`updateZoomOutButton` and `updateCountySelectorDropdown`)? it seems to me that it could be naturally paired with the test at line 175 (making that one into an `else if`).